README.md

```diff
@@ -1 +1,108 @@
+
+Si vous arrivez à trouver ce membre, il vous suffira de vous connecter à Kibana avec un login formé de la première lettre de son prénom, de la première lettre de son nom de famille et des deux derniers chiffres de son année de naissance. Son mot de passe sera celui qui est en clair dans le document trouvé dans l'index `members`.
+
+### Les opérations
+
+A partir d'ici, il faudra que vous vous connectiez avec les login et mot de passe du membre "Planificateur".
+
+J'ai réussi à récupérer le snapshot d'un index contenant un historique de toutes les opérations effectuées par `4STIC` dans le cadre de leur plan de grande envergure.
+
+Une fois connecté avec le user du "Planificateur", il vous faudra créer une repository (appelez le `privileged_repository` par exemple) de type Shared FS et pointant sur le répertoire `/mnt/data/privileged`.
+
+Vous trouverez alors dans ce repository un unique snapshot qu'il vous faudra restaurer (**sans le global state!**) pour avoir accès aux données des `operations`. De plus, ce snapshot est associé à des `metadata` qui vous donneront une autre partie de l'adresse email !
+
+Chacune de ces opérations contient les champs suivants :
+
+* `@timestamp` : la date d'exécution de l'opération
+* `type` : le type de l'opération
+* `city` : la ville concernée par l'opération
+* `login`: le login Elastic du membre qui a mené l'opération
+* `successful` : un booléen indiquant si l'opération a été un succès ou pas
+
+Cet index `operations` est basé sur un `index template`, il y en a plusieurs possibles mais un seul a été appliqué. Une fois découvert, vous trouverez une autre partie de l'adresse email dans les `metadata` associées à cet `index template`.
+
+Le chef de la sécurité de `4STIC` est un petit malin doublé d'un blagueur, il a disséminé des indices dans les données mais je ne manque pas de ressource et j'ai réussi à mettre la main sur la liste des indications permettant d'identifier le membre qui va être en charge de la prochaine opération. Rappelez-vous : il s'agit du cerveau de l'organisation, il est crucial d'obtenir cette information !
+
+#### Chaud
+
+Le premier indice ne peut pas être découvert à l'oeil nu, il faut observer les données sous un angle nouveau.
+
+Votre mission sera de :
+
+* Créer un Data View dans Kibana pour être capable d'observer les données de l'index `operations`
+* Toujours dans Kibana, créer une visualisation de type `Heatmap` dont les caractéristiques sont les suivantes :
+  * L'axe des abscisses affiche les `type` d'opérations, classés par ordre alphabétique (attention, il y en a 8 en tout, nous voulons tous les voir !)
+  * L'axe des ordonnées affiche les `city` dans lesquelles se sont déroulées les opérations (idem, il y en 8 !)
+  * La troisième dimension sera le nombre d'opérations : il faudra choisir une échelle de couleur en dégradé, par exemple `Cool` ou `Warm`
+
+Si votre visualisation est bien conçue, vous devriez voir apparaître un chiffre !
```

Comment:
En suivant les instructions, j'arrive sur ça:
![image](https://user-images.githubusercontent.com/191606/182122203-37e998da-5378-4a1d-aefc-cf8b58d803e1.png)
Pourtant j'ai tout trié par ordre alphabétique. Pour avoir la bonne visu, j'ai dû coché "Group by this field first" sur l'axe vertical.